styles/main.css

```diff
@@ -534,3 +534,203 @@ form .cta-button:hover {
   margin-bottom: 0;
   text-align: center;
 }
+
+/* ##### DESKTOP DESIGN ##### */
+
+/* ##### DESKTOP DESIGN ##### */
+
+/* ##### DESKTOP DESIGN ##### */
+
```

Comment:
- [ ] To make your design looks very similar to the Figma template, Please add the margin-left and margin-right of your works section in the desktop version. you can do it easily by adding the padding property of the `class or id portfolio in your section <--PORTFOLIO-->`